N1C WAF Security Dashboard Release (#1834)
nginx/documentation: content/nginx-one-console/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## April 7, 2026

### Observability: F5 WAF for NGINX security dashboard

> A new Security Monitoring module provides real-time visibility into F5 WAF for NGINX security events. Monitor attack counts, violation types, and triggered signatures across your WAF instances with customizable dashboards and global filters.

Reviewer: ```suggestion
The new Security Monitoring module gives you real-time visibility into F5 WAF for NGINX security events. You can monitor attack counts, violation types, and triggered signatures across your WAF instances by using customizable dashboards and global filters.
```

> The new Security Monitoring module gives you real-time visibility into F5 WAF for NGINX security events. You can monitor attack counts, violation types, and triggered signatures across your WAF instances by using customizable dashboards and global filters.

Reviewer 2: can we also add a link here? seems like it would help?
